README.md

```diff
@@ -42,6 +65,6 @@ TBD.
 This repository uses Dagster Cloud Hybrid architecture with GitHub Actions to provide CI/CD.
 - The main branch is deployed to Dagster Cloud using the workflow in `.github/workflows/`. Each commit a new Docker image is built and pushed to our container registry. These images are deployed into an EKS cluster by our running Dagster Agent which also syncronizes changes with Dagster Cloud.
 
-- The open PR in this repository shows how Dagster supports full integration testing with a *branch deployment*, in this case the PR is code for a second "competing" model. This change also highlights how you can test dependency changes. This cxapability is also implemented in the GitHub Action in this repo.
+- The open PR in this repository shows how Dagster supports full integration testing with a *branch deployment*, in this case the PR is code for a new dbt model.
 
-*Dev Notes in the Repo Wiki*
+*Additional Dev Notes in the Repo Wiki*
```

Comment:
do we have any notes in the Wiki now? Or is this aspirational?

Reply:
we do! https://github.com/dagster-io/hooli-data-eng-pipelines/wiki/Development-Notes

(though they might be somewhat out of date)